timescaledb/how-to-guides/hypertables/distributed-hypertables.md

```diff
@@ -291,15 +291,40 @@ the data over the data nodes, similar to traditional sharding.
 For more information about partitioning distributed hypertables, see the
 [About multi-node section][about-multinode].
 
+## Repartitioning distributed hypertables
+You can expand distributed hypertables by adding additional data nodes. If you 
+now have fewer space partitions than data nodes, you need to increase the
+number of space partitions to make use of your new nodes. The new partitioning
+configuration only affects new chunks. In this diagram, an extra data node 
+was added during the third time interval. The fourth time interval now includes 
+four chunks, while the previous time intervals still include three:
+
+<img class="main-content__illustration" src="https://s3.amazonaws.com/assets.timescale.com/docs/images/repartitioning.png" alt="Diagram showing repartitioning on a distributed hypertable"/>
```

Comment:
Text looks great but doesn't seem to entirely match the diagram. It says the newer configuration requires data from three chunks but only two are highlighted in the picture.